### PR TITLE
Integrate translation checker in validation process

### PR DESCRIPTION
**Pull request by mkangia**
Building on https://github.com/dimagi/CommcareTranslationChecker/pull/15
https://dimagi-dev.atlassian.net/browse/ICDS-393

best review by commit
added label to wait for checker version update on PyPi

### CODE REVIEW

File: corehq/apps/translations/app_translations.py [hunk 1]
```diff
@@ -6,11 +6,13 @@
 import itertools
 import re
 import ghdiff
+import openpyxl
```
Comment (emord) [inline corehq/apps/translations/app_translations.py]: nit: imports should generally follow https://www.python.org/dev/peps/pep-0008/#imports meaning ghdiff and openpyxl would go where six is and io would rise up here. (would be nice to get isort functionality for stickler)

File: corehq/apps/translations/app_translations.py [hunk 2]
```diff
@@ -191,29 +196,63 @@ def process_bulk_app_translation_upload(app, workbook):
     return msgs
 
 
-def validate_bulk_app_translation_upload(app, workbook, email):
+def run_translation_checker(file_obj):
+    translation_checker_messages = []
+    result_wb = None
+    try:
+        result_wb = validate_workbook(openpyxl.load_workbook(file_obj), translation_checker_messages)
```
Comment (emord) [inline corehq/apps/translations/app_translations.py]: validate_workbook comes from the library,and the library requires openpyxl. would it be possible to just provide a file object and it loads the workbook for you?

**Review by emord — APPROVED** [review]

File: corehq/apps/translations/app_translations.py (hunk 2 above)
Comment (mkangia) [reply corehq/apps/translations/app_translations.py]: https://github.com/dimagi/CommcareTranslationChecker/pull/16

File: corehq/apps/translations/app_translations.py (hunk 1 above)
Comment (mkangia) [reply corehq/apps/translations/app_translations.py]: thanks for pointing that. Will update.

**Review by emord — APPROVED** [review]

File: corehq/apps/translations/app_translations.py [hunk 3]
```diff
@@ -209,25 +211,47 @@ def validate_bulk_app_translation_upload(app, workbook, email, file_obj):
...
         return [(messages.error, _("Issues found. You should receive an email shortly."))]
     else:
         return [(messages.success, _("No issues found."))]
 
 
-def _email_app_translations_discrepancies(msgs, email, app_name):
-    html_content = ghdiff.default_css
-    for sheet_name, msg in msgs.items():
-        html_content += "<strong>{}</strong>".format(sheet_name) + msg
+def _email_app_translations_discrepancies(msgs, text_messages, email, app_name, result_wb):
```
Comment (esoergel) [inline corehq/apps/translations/app_translations.py]: nit: you could use the same variable name here and above (text_messages vs checker_messages) for simplicity

**Review by esoergel — APPROVED** [review]

**Pull request merged**